OrbitQt/orbitmainwindow.cpp

```diff
@@ -212,6 +212,14 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App,
     OnTimerSelectionChanged(timer_info);
   });
 
+  GOrbitApp->SetTimerSelectedCallback([this](const orbit_client_protos::TimerInfo* timer_info) {
+    OnTimerSelectionChanged(timer_info);
+  });
+
+  GOrbitApp->SetTimerSelectedCallback([this](const orbit_client_protos::TimerInfo* timer_info) {
+    OnTimerSelectionChanged(timer_info);
+  });
+
```

Comment:
Is this intended to be part of the change?

Reply:
Ugh... no, this looks like a rebasing mistake.

Reply:
Removed.